### PR TITLE
Ryan Signup

### PR DESCRIPTION
**Pull request by not-ryan-ning**
NOTE: Empty ArrayLists are initialized for the new users, for profile, playlist and their friends and requests.

### CODE REVIEW

**Comment by not-ryan-ning** [comment]
This branch is ready to merge, interface adapters NOT implemented - conflicting issues with ViewManagerModel - Can someone review the Playlist Entities

**Comment by not-ryan-ning** [comment]
I've reviewed all the conflicts, if someone can review and merge that would be great

File: src/entity/CommonPlaylist.java [hunk 1]
```diff
@@ -59,8 +59,5 @@ public double getInstrumentalness() {
     public double getValence() {
         return valence;
     }
-    @Override
-    public ArrayList<String> getTopThreeArtists() {
```
Comment (ShihHsin0723) [inline src/entity/CommonPlaylist.java:63]: CommonPlaylist entity includes the topThreeArtists attribute, so we should include this getter method.

**Review by ShihHsin0723 — APPROVED** [review]
Looks good!

**Pull request merged**